spec/declaration.dd

```diff
@@ -502,6 +502,44 @@ a = 2;         // sets `S.i` to `2`
 b = 4;         // sets `S.j` to `4`
 -----------
 
+
+$(H2 $(LNAME2 alias-reassignment, Alias Reassignment))
+
+        $(GRAMMAR
+        $(GNAME AliasReassignment):
+            $(GLINK_LEX Identifier) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK Type)
+            $(GLINK_LEX Identifier) $(D =) $(GLINK2 expression, FunctionLiteral)
+            $(GLINK_LEX Identifier) $(D =) $(GLINK StorageClasses)$(OPT) $(GLINK BasicType) $(GLINK2 function, Parameters) $(GLINK2 function, MemberFunctionAttributes)$(OPT)
```

Comment:
Sorry, how is this different from the `AliasAssign` rule added in #2868?

Furthermore, `AliasReassignment` is not referenced anywhere in the spec, which effectively makes this syntax not a part of the language. What does this grammar block describe exactly?

@WalterBright

Reply:
@WalterBright ping

Reply:
@WalterBright ping